### DISCUSSION TITLE
Transfer an existing project to another organization without having owner role

### DISCUSSION THREAD

**Discussion by workhardforcoding** [supabase/supabase#27972] [Questions]
Dear community, one of our employees in our startup left and is trying to not hand over the ownership of Supabase. We asked for the owner role to safely transfer the project but he didn't respond to our message. I'm wondering how to transfer a project without having the owner role. I also opened a support ticket for this but haven't got any response yet. Thanks

**Comment by GaryAustin1** [comment]
Support is likely the only way to resolve transferring.
You could do a migration though I think by copying the data to a new instance.
